presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcStorageManager.java

```diff
@@ -570,6 +570,16 @@ public void rollback()
                 for (File file : stagingFiles) {
                     file.delete();
                 }
+
+                // cancel incomplete backup jobs
+                futures.forEach(future -> future.cancel(true));
+
+                if (backupStore.isPresent()) {
+                    // delete completed backup shards
+                    for (ShardInfo shard : shards) {
+                        backupStore.get().deleteShard(shard.getShardUuid());
```

Comment:
This should use `BackupManager` and wait for the result so that we don't overload the backup store.